setup.cfg

```diff
@@ -36,6 +36,7 @@ install_requires =
     shiny
     click>=8.1.7
     appdirs>=1.4.4
+    lzstring>=1.0.4
```

Comment:
I added `lzstring` here but the checks are still reporting "no module named 'lzstring'". I can't repro this locally, `pip install -e ".[dev,test]"` installs lzstring.